.github/workflows/update.yml

```diff
@@ -0,0 +1,31 @@
+name: Check Update
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: "0 0 * * *"
```

Comment:
Currently the watchdog runs every 5 min (https://github.com/factoriotools/factorio-docker-watchdog/blob/master/factorio-docker-watchdog.go#L23) would it be possible increase the check interval?
Would every hour be enough? i think once per day is not enough.

Reply:
once an hour seems already enough, not really need 5 minutes